src/views/Quizz/Quizz.vue

```diff
@@ -14,20 +14,36 @@
         :key="'content' + index"
         :step="step"
       )
-        v-card.mb-12(
-          color="grey lighten-1"
-          height="200px"
-        )
-      v-btn(color="primary" @click="nextQuestion()") Valider
+        span {{ 'Question ' + (index + 1) + ' : ' + questionGroup.question }}
```

Comment:
I think you can use interpolation here `Question  ${index++}  :   ${questionGroup.question}`